components/script/dom/resizeobserver.rs

```diff
@@ -97,52 +97,48 @@ impl ResizeObserver {
         can_gc: CanGc,
     ) {
         let mut entries: Vec<DomRoot<ResizeObserverEntry>> = Default::default();
-        let observation_targets = self.observation_targets.borrow();
-        for (observation, target) in observation_targets.iter() {
-            let observation_state = observation.state.borrow();
-            let ObservationState::Active(box_size) = &*observation_state else {
-                continue;
-            };
+        for (observation, target) in self.observation_targets.borrow().iter() {
+            if let ObservationState::Active(box_size) = &*observation.state.borrow() {
```

Comment:
Oh, shoot, I misread the original code when I made my suggestion. I meant this:
```rust
let ObservationState::Active(box_size) = &*observation.state.borrow() else {
  continue;
};
```
which avoids the need to indent all the code after it.

Reply:
oop no problem, let me revert that